content/articles/setup-and-run-applications-using-typescript-with-aureliajs/index.md

```diff
@@ -0,0 +1,270 @@
+
+This guide will help you set up and run Applications using TypeScript while still using the Aurelia.js framework.
+
+### Why use TypeScript
+
+On the other hand, Typescript is a superset of JavaScript. Typescript is gaining a lot of popularity among JavaScript developers. It is one of the fast-developing programming languages for building extensive applications.
+
+### How to set up a basic TypeScript Aurelia project
+
+Let's dive and set up Aurelia TypeScript. First, you need to install the [Aurelia CLI](https://aurelia.io/docs/cli/webpack/) (command-line interfaces). An interface is a medium through which a user can interact with any tool using a computer. A CLI is a way to interact with the computer program bY typing in instructions or commands.
+
+Aurelia provides CLI commands that allow you to bootstrap the Aurelia project. This allows you to have a sample project that you can build on rather than creating an application from scratch. This bootstrapped app comes will preconfigured settings that you can use on the fly. This prevents developers from creating redundant settings every time they create an appliv=cation. It saves time and increases the developers' productivity.
+
+To bootstrap an Aurelia app, you must first install the [Aurelia CLI](https://aurelia.io/docs/cli/webpack/) globally on your computer. To do this, run this command on your terminal:
+
+```bash
+npm install aurelia-cli -g
+```
+
+Once you have installed the CLI, proceed and bootstrap the basic Aurelia project. Create a project folder and run this Aurelia command:
+
+```bash
+au new
+```
+
+This command will launch an interactive interface that allows you to choose the default settings of your Aurelia project. Once you run the command, enter the application name of your choice.
+
+![auleria-app](auleria-app.png)
+
+In this tutorial, we are creating the Auleria app with TypeScript code. Thus you need to set the right environment to execute TypeScript. In your command line, select the **Default TypeScript App** as shown below:
+
+![aurelia-typescript](aurelia-typescript.jpg)
+
+Use the arrow keys or numbers to select the **Default TypeScript App** and hit enter to submit.
+
+Finally, select the dependency manager you want to use between NPM and Yarn.
+
+This will create a ready configured Aurelia TypeScript app. To test if the setup is working, run `npm start`. Once done, open `http://localhost:8080/` on your browser. This will log a **Hello World** on your browser.
+
+![test-app](test-app.jpg)
+
+This indicates that the created setup is working and ready to start building on your Auleria applications.
+
+### Build a Todo app using Aurelia and TypeScript
+
+Let's dive in to create and run a todo application using this configuration. This will use the TypeScript code to create the application.
```

Comment:
Building a Todo application is becoming common these days. If possible, can we add something totally new? It helps you with SEO. Because, most of the introductory articles implements Todo application. When you build something new, the readers would definitely read yours.

Just a suggestion.

Reply:
Add what does the todo application do? What features does your application have? Any image of the expected output would help.